tests/Controllers/DevHealthControllerTest.php

```diff
@@ -37,4 +37,15 @@ public function testIndexCreatesChecker()
 
         $this->assertInstanceOf(EnvironmentChecker::class, $controller->index($request));
     }
+
+
+    public function testHealthDoesNotIncludeDetails()
```

Comment:
We should probably also have one that checks `DevCheckController` _does_ include the details if that's okay.

Reply:
Done